components/expenses/PayExpenseModal.js

```diff
@@ -252,35 +284,48 @@ const PayExpenseModal = ({ onClose, onSubmit, expense, collective, host, error }
               />
             </Amount>
           </AmountLine>
-          {formik.values.paymentProcessorFee !== null && (
+          {canQuote && quoteQuery.loading && (
+            <AmountLine borderTop="0.8px dashed #9D9FA3">
+              <Label>
+                <FormattedMessage id="PayExpense.ProcessorFeesInput" defaultMessage="Payment processor fees" />
+              </Label>
+              <Amount>
+                <LoadingPlaceholder height="16px" />
+              </Amount>
+            </AmountLine>
+          )}
+          {canQuote && paymentProcessorFee?.valueInCents && (
```

Comment:
@kewitz I'm not sure about the `canQuote` part of this condition, don't we want to surface the fees whenever they're there? I ended up in this situation while testing and was surprised to not see the fees in the breakdown.

![image](https://user-images.githubusercontent.com/1556356/145961722-386a6169-babb-4a5f-89c7-93428c171107.png)

Reply:
True. I'm fixing that.